### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by LaurentRouchairoles** [ataraxus/meshdoxer#2]
I tried to install but it fails to find the deb : https://s3.amazonaws.com/autodesk-meshmixer/meshmixer/amd64/meshmixer_2.9_amd64.deb
and I have a bunch of errors : 
"
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (This frontend requires a controlling tty.)
debconf: falling back to frontend: Teletype
"

**Comment by ataraxus** [comment]
Hey, thanks for the heads up! apparently autodesk has removed all traces of meshmixer for linux... I dont know if I would be allowed to distribute it in any way my self. If you know where to download it, i will update the Dockerfile but for now it wont work. Sorry

**Comment by LaurentRouchairoles** [comment]
Yes it seems autodesk does not want to deal with linux ! Would it be possible that you let a copy of meshmixer on your github account  so I can get it. We delete the comments and the file after !? For the errors it may be related to xterm ? (I don't know much about linux insides)

**Comment by LaurentRouchairoles** [comment]
Found that, it that may help improve future docker auto installation, no for meshmisher but an other program... http://forum.doozan.com/read.php?2,502,506 someone says the same type of errors are solved by installing "dialog" !? "apt-get install dialog"
